docs/src/z20-for-pkg-devs.md

```diff
@@ -191,6 +191,14 @@ It attempts to use webscraping and such to workout what should be in the registr
 You can then edit the generated code to make it suitable for your use.
 (E.g. remove excessive information in the message)
 
+### Removing data
+
+Sometimes you don't need the data anymore.
+You can remove files from within Julia using the `rm` command.
+If you had registered a `DataDep` called `MyDataName`, then you can remove it with
+```
+rm(datadep"MyDataName"; recursive=true)
+```
```

Comment:
Cool thanks.
I think this is probably better in the docs for end-users.

https://github.com/oxinabox/DataDeps.jl/blob/master/docs/src/z10-for-end-users.md

What do you think?